docgen/src/guides/v3-migration.md

```diff
@@ -308,6 +308,10 @@ With the redo button:
 </div>
 ```
 
+### NumericSelector
+
+Widget removed.
```

Comment:
should say "replaced by `numericMenu`" or something like that

Reply:
It's not replaced but rather can be reproduced with NumericMenu. Let's write a guide later on?

Reply:
good for me, but should at least link to it IMO

Reply:
I agree - will do another PR to review the whole migration guide. Let's move forward with this PR.